common/http/src/main/java/io/helidon/common/http/Http.java

```diff
@@ -1332,6 +1332,13 @@ public static final class Header {
          * Implementation-specific fields that may have various effects anywhere along the request-response chain.
          */
         public static final HeaderName PRAGMA = HeaderEnum.PRAGMA;
+        /**
+         * The {@code Proxy-Connection} header name.
+         * Implemented as a misunderstanding of the HTTP specifications. Common because of mistakes in
+         * implementations of early HTTP versions. Has exactly the same functionality as standard
+         * Connection field. Must not be used with HTTP/2.
+         */
+        public static final HeaderName PROXY_CONNECTION = HeaderEnum.PROXY_CONNECTION;
```

Comment:
Can you please move this constant to Proxy class itself? I would prefer not to add bad header constants to the `HeaderEnum` (there is memory implication for htis).

Reply:
Done